Mathlib/Data/Finset/Lattice.lean

```diff
@@ -1383,7 +1383,7 @@ theorem max_of_nonempty {s : Finset α} (h : s.Nonempty) : ∃ a : α, s.max = a
 theorem max_eq_bot {s : Finset α} : s.max = ⊥ ↔ s = ∅ :=
   ⟨fun h ↦ s.eq_empty_or_nonempty.elim id fun H ↦ by
       obtain ⟨a, ha⟩ := max_of_nonempty H
-      rw [h] at ha; cases ha; done, -- Porting note: error without `done`
+      rw [h] at ha; cases ha; , -- Porting note: error without `done`
```

Comment:
As before, I'm not sure you should be touching this if you're not solving the porting note

Reply:
I think that the porting note is simply pointing out that a `,` does not stop the `cases` pattern.  This is the [Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/error.20puzzle/near/426043583).  So, I am going to remove the porting note!